### PR TITLE
MRG: Packagify bayes meeg

### LINKED ISSUE #1: make proper project with setup.py and use sphinx/sphinx-gallery to make a documentation.
@yousrabk maybe @jasmainak can give you a hand in exchange of some French lessons...

the project should mimic as much as possible 

https://autoreject.github.io/

@yousrabk do you think you can give it a try?

### PR DESCRIPTION
**Pull request by jasmainak**
closes https://github.com/yousrabk/bayes_meeg/issues/1

### CODE REVIEW

**Comment by jasmainak** [comment]
@yousrabk I cannot plot the example `plot_sample_circular_brain.py`: 

```py
OSError: No such file /home/mainak/Desktop/projects/github_repos/mne-python/examples
/MNE-sample-data/subjects/sample/label/lh.aparc.split-125.annot, 
candidate parcellations in that directory: lh.BA.annot, rh.BA.annot, rh.aparc.annot, 
rh.aparc.a2009s.annot, aparc.annot.ctab, lh.aparc.annot, lh.aparc.a2009s.annot, 
aparc.annot.a2009s.ctab
```
I tried updating my sample data but it still does not work. How do I get this file?

File: examples/plot_run_demo_solver.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
+"""
+=========
+Plot demo
+=========
+
+Plot demo.
```
Comment (jasmainak) [inline examples/plot_run_demo_solver.py]: The docstring here is necessary to make `sphinx_gallery` happy. I put dummy text in here. But happy to change it to whatever you like.

**Comment by yousrabk** [comment]
Thanks @jasmainak! For the `aparc.split-125`, it is a more precise parcellation that I have got from Alex. I will change it for let's say `aparc.a2009s` for now.

File: examples/plot_run_demo_solver.py (hunk 1 above)
Comment (yousrabk) [reply examples/plot_run_demo_solver.py]: Fixed

**Comment by jasmainak** [comment]
@yousrabk this is good to go from my end. After merging, you should do the following:

```sh
$ python setup.py develop
```
and then 

```
$ cd doc/
$ make install
```

and then check if the new website looks good. Ping me if you have problems :-)

**Pull request merged**